staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1531,6 +1539,8 @@ message PullImageResponse {
     // Reference to the image in use. For most runtimes, this should be an
     // image ID or digest.
     string image_ref = 1;
+    // Mount references the OCI volume mount of the pull image response.
+    OCIVolumeMount mount = 2;
```

Comment:
Since we embed the image_ref here, why can't we just have this be 
```
    // Reference to the image in use. For most runtimes, this should be an
    // image ID or digest.
    string image_ref = 1;
   // Specifies the value of the OCI mount.
    string mount_ref = 2;
```
if the mount_ref is "", then the kubelet knows it was pulled as an image_ref, and can reference that to do the second mount. 

Ultimately it feels odd to me to specify image_ref twice. WDYT @mikebrow @saschagrunert

Reply:
or another option is we specify one image_ref and then we embed ImageRefType enum with the below types

Reply:
I assume we can just add a mount_ref, right @mikebrow?

Reply:
the reason for type.. would be to avoid versioning issues.